tests/quic-alerts/test.yaml

```diff
@@ -0,0 +1,39 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+  min-version: 6.0.0
```

Comment:
I think we want 7 as min-version

Reply:
happy new year and ping @sharksforarms can you do this little change ?

Reply:
@catenacyber happy new year! done :)

Reply:
thanks